src/services/pmo/api/api.ts

```diff
@@ -30,6 +52,13 @@ export async function apiRequest<
       ? null
       : new Error(`${response.status} (${response.statusText})`)
 
+    if (method === "GET" && response.ok) {
+      storeData<LocalStorageApiRequest<Data>>(`${baseUrl}${path}?${query}`, {
+        data: data,
+        dateTime: new Date(),
```

Comment:
I would consider storing `new Date().valueOf()` so you don’t have to create a `Date` object above.